File: app/src/main/kotlin/ru/ktoddler/model/entity/KToddlerEntity.java
package ru.ktoddler.model.entity;

import android.support.annotation.NonNull;

import nl.qbusict.cupboard.annotation.Column;
import ru.ktoddler.model.entity.contract.Contract;

public class KToddlerEntity {

    @Column(Contract.ID)
    public long id;

    @NonNull
    @Column(Contract.KToddlerEntity.NAME)
    public String name = "";

    @Column(Contract.KToddlerEntity.AGE)
    public int age = 1;
}

File: app/src/main/kotlin/ru/ktoddler/model/entity/KToddlerEntity.kt
package ru.ktoddler.model.entity

import nl.qbusict.cupboard.annotation.Column
import ru.ktoddler.model.entity.contract.Contract

class KToddlerEntity {

    @Column(Contract.ID)
    var id: Long = 0

    @Column(Contract.KToddlerEntity.NAME)
    var name = ""

    @Column(Contract.KToddlerEntity.AGE)
    var age = 1

}
